addons/overpressure/functions/fnc_firedEHBB.sqf

```diff
@@ -18,6 +18,8 @@
 //IGNORE_PRIVATE_WARNING ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_gunner"];
 TRACE_8("firedEH:",_unit,_weapon,_muzzle,_mode,_ammo,_magazine,_projectile,_gunner);
 
+private _shooter = if (vehicle _unit != _unit) then {_gunner} else {_unit};
```

Comment:
not tested, but what if you shoot a rpg from a FFV slot?
ie, unit is in vic but _gunner would still not be defined?

Reply:
Good point.
Unfortunately shooting from a FFV position also leaves `_gunner` undefined.